### ISSUE TITLE
Bridge ignores bind errors completely on Windows

### ISSUE THREAD

**Issue by IngwiePhoenix** [ProtonMail/proton-bridge#295]
## Expected Behavior
The ability to connect to my E-Mail client.

## Current Behavior
This has plagued me for literal __months__. I did E-Mail support, was told that it'd be looked into, and I am really just at a point where I am tearing out my own hair.

```
ERRO[Sep 14 11:38:58.924] Cannot start listner.                         address="127.0.0.1:1143" error="listen tcp 127.0.0.1:1143: bind: Der Zugriff auf einen Socket war aufgrund der Zugriffsrechte des Sockets unzulÃ¤ssig." pkg=serverutil protocol=IMAP useSSL=false
```

Even though this error is obvious, the bridge does not report it, nor automatically adjusts nor suggests something else. Yes, permissions in the firewall are granted.

To be frank, this is annoying.

## Possible Solution
Show an error message or automatically adjust the port and let the user know.

## Steps to Reproduce
I installed, configured and then tried to use the bridge.

## Version Information
Windows: `Microsoft Windows [Version 10.0.19044.1949]`
Bridge: 2.3.0

## Context (Environment)
I am paying for something I can not use to it's full extend. Usually I end up just using the webmail client, but the main reason I decided to pay for my account was to use desktop mail clients to separate general browsing and E-Mail coordination.

## Detailed Description
This software just lives in the info tray and "exists". Unobtrusive and all, which is great. But if an error occurs, it should step into the foreground and let the user know. If I hadn't been changing ports and tailing logs, I'd have never known why my clients can not connect. This should not be a problem, at all.

## Possible Implementation
A notification on startup or a dialog. Should be switch-able by the user, perhaps.

**Comment by ericLemanissier** [comment]
I had the same experience, on a a computer having two user sessions opened at the same time: both bridges start and report no error, even if only the first bridge could actually "take" the port before the second bridge is started.

**Comment by andrzejsza** [comment]
we now have proper notifications for occupied ports, implemented in 2.4.3 beta. closing for now - let us know if still a problem.
